contracts/FlowServiceAccount.cdc

```diff
@@ -126,6 +143,11 @@ pub contract FlowServiceAccount {
             FlowServiceAccount.accountCreators.remove(key: accountCreator)
             emit AccountCreatorRemoved(accountCreator: accountCreator)
         }
+
+         pub fun setIsAccountCreationRestricted(_ enabled: Bool) {
```

Comment:
Should this maybe emit an event?

Reply:
I did not know adding an event type was legal. I will add this.

Reply:
added event emit.